plugin-server/src/config/config.ts

```diff
@@ -125,6 +125,9 @@ export function getDefaultConfig(): PluginsServerConfig {
             ? 1024 // NOTE: ~1MB in dev or test, so that even with gzipped content we still flush pretty frequently
             : 1024 * 50, // ~50MB after compression in prod
         SESSION_RECORDING_REMOTE_FOLDER: 'session_recordings',
+        // a different threshold to SESSION_RECORDING_MAX_BUFFER_AGE_SECONDS because we want
+        // more tolerance for idle chunks to avoid dropping chunks that will eventually be completed
+        SESSION_RECORDING_IDLE_CHUNKS_THRESHOLD_SECONDS: 60 * 120, // NOTE: 2 HOURS
```

Comment:
2 hours is huge when you think about it. The chunks are sent to kafka one after another so they should be within a few seconds of each other. Unless I'm completely missing something this value should be really low like a couple of minutes tops

Reply:
For the chunk in question I believe that we waited at least ten minutes and then dropped the first 14 chunks

So, I thought what value will help avoid lag but leave ample time for any chunks to have been delivered unless we're completely confused about what is happening.

If that pans out then we can either tune this value, or handle chunks differently. I'm very unconvinced by this idle dropping of chunks (which I was only forced to introduce to unblock other sources of lag)

Reply:
The flip of this is that pending chunks is holding like half a gig in RAM, so maybe we should drop it faster 😅